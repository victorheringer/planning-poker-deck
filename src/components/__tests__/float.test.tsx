import { render } from "@testing-library/react";
import Float from "../float";

describe("Float", () => {
  it("should render correctly", () => {
    const tree = render(<Float />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
