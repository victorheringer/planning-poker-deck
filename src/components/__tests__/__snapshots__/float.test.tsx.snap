// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Float should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL gnKGyL"
  />
</DocumentFragment>
`;
